packages/web-app-files/src/views/shares/SharedViaLink.vue

```diff
@@ -131,11 +155,6 @@ export default defineComponent({
     }
   },
 
-  async created() {
-    await this.loadResourcesTask.perform()
-    this.scrollToResourceFromRoute(this.paginatedResources)
-  },
-
```

Comment:
Sorry, I just meant removing the mutation subscription (like you did), not the entire `created` hook 😅

Reply:
Ah, sorry I don't where was my head, reverted and removed only the mutation :sweat_smile: